POLICIES.rdoc

```diff
@@ -45,14 +45,31 @@ For example, RubyGems 2.5.0 was recently released.  RubyGems does not
 guarantee any non-security bugfix releases will be made upon the 2.4.x series,
 or any earlier series, of releases.
 
-Additionally, when a Ruby version reaches end-of-life the following release of
-RubyGems will no longer be required to maintain backwards compatibility with
-that Ruby version.  This may result in a major version change.
+Additionally, when a Ruby version reaches end-of-life the following minor
+release of RubyGems will drop backwards compatibility with that Ruby version.
 
-For example, since Ruby 1.9.3 has reached end-of-life, RubyGems may use
-features of ruby that only exist in Ruby 2.0 and later.  As of this writing
-RubyGems is 2.5.x, so the RubyGems version would become 3.x if it used
-features only available in Ruby 2.0 and later.
+For example, since Ruby 2.2 has reached end-of-life, RubyGems may use require
+Ruby 2.3 or later.  As of this writing RubyGems is at versino 3.0, so the
+RubyGems version would become 3.1 if it used features only available in Ruby
+2.3 and later.
+
+=== Security Release Process
+
+- Obtain CVE numbers as needed from HackerOne or RedHat.
+- Agree on a release date with ruby-core, so patches can be backported to older
+  Ruby versions as needed.
+- Avoid releasing security updates on Fridays, so platform services don't have
```

Comment:
I think this is unfair to those who are actually doing the releases

Reply:
Is it? How so? Wouldn't you just wait until Monday?

Reply:
@segiddins can you elaborate on what part of this seems unfair to the person doing the release? The goal is definitely to come up with a solution that works for the maintainers, the releaser, and the downstream developers who need to deploy those releases.

Reply:
I oftentimes only have time to do releases (which take more than just a few minutes) on weekends or holidays. “Just wait until Monday” doesn’t really work

Reply:
That seems totally reasonable. We can take security releases on a case-by-case basis, and one of myself, @hsbt, @colby-swandale, or @claudijd can handle RubyGems security releases as needed. 👍

Reply:
I can work it instead of @segiddins in the day time.